engines/wasmer/example_test.go

```diff
@@ -0,0 +1,30 @@
+package wasmer
```

Comment:
Please add this build tag. I have an M1 Mac apparently this feature is not supported.

```go
//go:build amd64
// +build amd64
```